### PR TITLE
DOCS-1404: added-dtb-test

### PR DESCRIPTION
**Pull request by Alex-MultiSafepay**
### Basic
- We only accept documentation that is proved to work on our LIVE API
- At least 1 approval is needed before pull requests can be merged
- The article must match our Synonym word list. For external commits, a MultiSafepay employee will check
- US English must be applied
- Use bullets for numerations where the sort order doesn't matter. For chronological order, use numbers. This does not apply to changelogs
- Set link on email address.

### Text style
- Paths are written like: _Configuration → System → Plugin → MultiSafepay_
- Use single quotation marks in your Markdown code (example title: 'Title of the article')
- Use Italics for buttons, clicks, paths, etc. Do mind to close the italics before the end of a sentence followed by a period
- Numerations / bullets are punctuated with a period at the end of the numerations / bullets.
- Periods should be excluded for sentences that finish in an e-mail address (e.g. "..contact us at <example@example.com> "  )

### Images
- Screenshot recomended sizes: Width: 820px - Height: auto

### CODE REVIEW

File: content/faq/getting-started/test-payment-details.md [hunk 1]
```diff
@@ -116,6 +116,19 @@ MultiSafepay provides a test platform for SEPA Direct Debit transactions. By ent
 | NL87ABNA0000000003 | Initialized/Uncleared/Completed | Transaction is Initialized ( after 2 minutes it is Uncleared, and 1 minute later it is Completed) |
 | NL87ABNA0000000004 | Initialized/Uncleared/Declined | Transaction is Initialized ( after 2 minutes it is Uncleared, and 1 minute later it is Declined) |
 
+## Direct Bank Transfer
+
+MultiSafepay provides a test platform for Direct Bank Transfer transactions though Deutsche Bank.
+You can simulate those scenarios:
```
Comment (Atiq-MultiSafepay) [inline content/faq/getting-started/test-payment-details.md:122]: You can simulate the following scenarios

File: content/faq/getting-started/test-payment-details.md [hunk 2]
```diff
@@ -116,6 +116,19 @@ MultiSafepay provides a test platform for SEPA Direct Debit transactions. By ent
 | NL87ABNA0000000003 | Initialized/Uncleared/Completed | Transaction is Initialized ( after 2 minutes it is Uncleared, and 1 minute later it is Completed) |
 | NL87ABNA0000000004 | Initialized/Uncleared/Declined | Transaction is Initialized ( after 2 minutes it is Uncleared, and 1 minute later it is Declined) |
 
+## Direct Bank Transfer
+
+MultiSafepay provides a test platform for Direct Bank Transfer transactions though Deutsche Bank.
```
Comment (Atiq-MultiSafepay) [inline content/faq/getting-started/test-payment-details.md:121]: You can simulate the following scenarios

File: content/faq/getting-started/test-payment-details.md [hunk 3]
```diff
@@ -116,6 +116,19 @@ MultiSafepay provides a test platform for SEPA Direct Debit transactions. By ent
...
+You can simulate those scenarios:  
+
+| Status    | Description              |
+| --------- | ------------------------ |
+| Completed | Transaction is Completed |
+| Canceled | Transaction is Canceled |
+
+Select "Demo Bank" in the Bank field and go through the steps with the information provided in the description in order to get a Completed status.
+In order to get a Cancel status you just need to click on the close button on the top right of the screen.
```
Comment (Atiq-MultiSafepay) [inline content/faq/getting-started/test-payment-details.md:130]: In order to get a _Cancel_ status you need to click on the _Close_ button at the top right of the screen.

File: content/faq/getting-started/test-payment-details.md [hunk 4]
```diff
@@ -116,6 +116,19 @@ MultiSafepay provides a test platform for SEPA Direct Debit transactions. By ent
...
+
+MultiSafepay provides a test platform for Direct Bank Transfer transactions though Deutsche Bank.
+You can simulate those scenarios:  
+
+| Status    | Description              |
+| --------- | ------------------------ |
+| Completed | Transaction is Completed |
+| Canceled | Transaction is Canceled |
+
+Select "Demo Bank" in the Bank field and go through the steps with the information provided in the description in order to get a Completed status.
```
Comment (Atiq-MultiSafepay) [inline content/faq/getting-started/test-payment-details.md:129]: Select 'Demo Bank' in the Bank field and go through the steps with the information provided in the description in order to get a _Completed_ status.

Comment (Alex-MultiSafepay) [reply content/faq/getting-started/test-payment-details.md]: Thanks @Atiq-MultiSafepay i've committed the new changes

**Pull request merged**